step2-05/demo/README.md

```diff
@@ -76,27 +76,25 @@ Given a state tree shaped like:
 }
 ```
 
-We would organize our reducers matching the keys of the state tree:
+We would organize our reducers matching the keys of the state tree and combine them with [`combineReducers()`](https://redux.js.org/recipes/structuring-reducers/using-combinereducers):
 
 ```ts
 import { createReducer } from 'redux-starter-kit';
-import { combineReducer } from 'redux';
+import { combineReducers } from 'redux';
 
-const reducer = combineReducer({
-  // first argument: initial state
```

Comment:
Realized this comment distracts from the main point of this section, which is combineReducers